app/tests/unit/components/metamineNU/neighbor-table.spec.js

```diff
@@ -0,0 +1,50 @@
+import createWrapper from '../../../jest/script/wrapper'
+import { enableAutoDestroy, mount, createLocalVue } from '@vue/test-utils'
+import NeighborPanel from '@/pages/metamine/visualizationNU/components/NeighborPanel.vue'
+import Vuex from 'vuex'
+import VueMaterial from 'vue-material'
+import { mockDataPoint } from './constants'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+localVue.use(VueMaterial)
```

Comment:
You shouldn't need to import Vuex and VueMaterial in the unit tests or create a local vue, were you getting specific errors related to this? For things that require store you can usually use `import store from '@/store'` and mock any commits/etc.